From 2f731ddc9fdc0bb3737d83dccb627c1f8aa68ba9 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Tue, 3 Jul 2018 19:50:21 +0800
Subject: [PATCH 1334/1345] arm64: dts: marvell: disable spi0 dts node for
 armada-70400-db platform

Disable the spi0 dts node in ap806 config space. Because the nor-flash
is not connected with spi0 signal pin. Some resistances are not welded
on armada-70400-db platform.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index 7d9998a..01b6755 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -228,7 +228,7 @@
 };
 
 &spi0 {
-	status = "okay";
+	status = "disabled";
 
 	spi-flash@0 {
 		#address-cells = <1>;
-- 
1.7.9.5

